libs/fetcher.client.js

```diff
@@ -187,17 +181,19 @@ function executeRequest(request, resolve, reject) {
         },
         request._clientConfig
     );
-    var headers = config.headers || request.options.headers || {};
+    options.config = config;
+    options.headers = config.headers || request.options.headers || {};
 
-    var url;
     var baseUrl = config.uri;
     if (!baseUrl) {
         baseUrl = config.cors
             ? request.options.corsPath
             : request.options.xhrPath;
     }
 
-    if (request.operation === OP_READ) {
+    if (request.operation === OP_READ && !config.post_for_read) {
```

Comment:
post for read is weird, I know this is old, what is the use case for this again?

Reply:
To be really honest, I don't know. I see in my code base that we used to use `create` instead of `read` just to make sure we would have a POST. I think the reason for that was related to how our bot protection used to work. In that case, calling `read` with `post_for_read` would be semantically better.

Reply:
By the way, I checked that the support exists since the very first implementation. Don't you have any usage of it in your code base?